packages/account/src/Components/trading-assessment/trading-assessment-new-user.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import TradingAssessmentForm from './trading-assessment-form';
+
+const TradingAssessmentNewUser = ({
+    assessment_questions,
+    goToNextStep,
+    goToPreviousStep,
+    onSave,
+    onCancel,
+    onSubmit,
+    getCurrentStep,
+    value,
+}) => {
+    const handleCancel = values => {
+        const current_step = getCurrentStep() - 1;
+        onSave(current_step, values);
+        onCancel(current_step, goToPreviousStep);
+    };
+
+    const handleSubmit = (values, actions, should_override) => {
+        let process_form_values = { ...values };
+        if (should_override) {
+            // Remove the keys with no values
+            process_form_values = Object.entries(process_form_values).reduce((accumulator, [key, val]) => {
+                if (val !== '') {
```

Comment:
Can we do this instead?

```suggestion
                if (val) {
```